src/ReactiveDomain.Testing/Specifications/NullReader.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Ignore Spelling: Tos
```

Comment:
What's this? I see it in a few files here.

Reply:
leftovers from spelling error detection in PowerModels project settings

Reply:
referencing LinkTos